docs/collections.md

```diff
@@ -10,24 +10,22 @@ Then run the following command:
 ./createcollection -c <collection_name>
 ```
 
-The script will ask you to run additional commands if it ran succesfully:
+The script will ask you to run additional commands if it ran successfully:
 ```shell
-# run migrations
-docker-compose run --rm snoop--<collection_name> ./manage.py migrate
-# reset stats index
-docker-compose run --rm snoop--<collection_name> ./manage.py resetstatsindex
-# create collection inside the snoop docker image
-docker-compose run --rm snoop--<collection_name> ./manage.py createcollection <collection_name> /opt/hoover/collections/<collection_name>/data
-# add collection to search
-docker-compose run --rm search ./manage.py addcollection <collection_name> http://snoop--<collection_name>/collections/<collection_name>/json --public
-# add ocr source
-docker-compose run --rm snoop--<collection_name> ./manage.py createocrsource ocr-name /opt/hoover/collections/<collection_name>/ocr-data
+1. Run the following command to start added containers using docker-compose
+  $ docker-compose up -d
+
+2. Run this command to wait for PostgreSQL startup:
```

Comment:
The instructions would be easier to read if the numbered list was a markdown list instead of a big box of code.

1. something
    ```shell
    rm -rf ~
    ```